### PR TITLE
Raise grpc max message size from 4M to 16M

### PR DESCRIPTION
**Pull request by bigclouds**
Raise grpc max message size from 4M to 16M

Signed-off-by: longguang.yue <yuelg@chinaunicom.cn>

### CODE REVIEW

File: pkg/plugin/nvidia/const.go [hunk 1]
```diff
@@ -35,4 +35,11 @@ const (
 	TotalGPUResource = "VOLCANO_GPU_TOTAL"
 	// NVIDIA visible devices
 	VisibleDevice = "NVIDIA_VISIBLE_DEVICES"
+
+	// DefaultMaxRecvMsgSize defines the default maximum message size for
+	// receiving protobufs passed over the GRPC API.
+	DefaultMaxRecvMsgSize = 16 << 20
+	// DefaultMaxSendMsgSize defines the default maximum message size for
+	// sending protobufs passed over the GRPC API.
+	DefaultMaxSendMsgSize = 16 << 20
```
Comment (hzxuzhonghu) [inline pkg/plugin/nvidia/const.go:44]: Without setting this the default value of send msg size of MaxInt32

Comment (hzxuzhonghu) [reply pkg/plugin/nvidia/const.go:44]: So the kubelet reporting error is caused by the grpc default recv msg size(4MB)

The kubelet side recv max limit

**Review by hzxuzhonghu — COMMENTED** [review]
I donot think the fix here can help, unless I miss something

**Comment by bigclouds** [comment]
@hzxuzhonghu It is kubelet's problem.
I see containerd has raised both size to 16M.  so this patch does the same thing.

**Comment by hzxuzhonghu** [comment]
Let's come back to device manager here, it will never recv a message exceed 1KB from kubelet based on the implementation.

 It can send response of any size, as the send message has no limitation. I am not sure this is necessary.

**Pull request closed without merging**